src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,21 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  return (
+    <div className="movies">
+      {
+        movies.map(
+          (elementOfMovies: Movie) => <MovieCard movie={elementOfMovies} />,
+        )
+      }
+    </div>
```

Comment:
```suggestion
    <div className="movies">
      {movies.map((movie: Movie) => (
        <MovieCard movie={movie} />
      ))}
    </div>
```
![image](https://user-images.githubusercontent.com/96409563/216920379-e424ed38-236f-4e37-be1a-fcd95ca08561.png)